build.gradle

```
@@ -73,6 +75,29 @@ dependencies {
     runtimeOnly "org.hsqldb:hsqldb"
 
     javaagent  libraries["splunk-otel-agent"]
+
+    jacocoAggregation project(':api')
+    jacocoAggregation project(':clients-core')
+    jacocoAggregation project(':clients:rh-marketplace-client')
+    jacocoAggregation project(':clients:product-client')
+    jacocoAggregation project(':clients:prometheus-client')
+    jacocoAggregation project(':clients:rbac-client')
+    jacocoAggregation project(':clients:quarkus:rbac-client')
+    jacocoAggregation project(':clients:rhsm-client')
+    jacocoAggregation project(':clients:subscription-client')
+    jacocoAggregation project(':clients:quarkus:subscription-client')
+    jacocoAggregation project(':clients:swatch-internal-subscription-client')
+    jacocoAggregation project(':clients:swatch-internal-subscription-springboot-client')
+    jacocoAggregation project(':clients:user-client')
+    jacocoAggregation project(':clients:rh-partner-gateway-client')
+    jacocoAggregation project(':clients:contracts-client')
+    jacocoAggregation project(':swatch-common-config-workaround')
+    jacocoAggregation project(':swatch-common-resteasy')
+    jacocoAggregation project(':swatch-core')
+    jacocoAggregation project(':swatch-system-conduit')
+    jacocoAggregation project(':swatch-producer-aws')
+    jacocoAggregation project(':swatch-contracts')
+    jacocoAggregation project(':swatch-product-configuration')
```

I dislike maintaining this list. I did a little bit of quick experimentation and came up with a way of having gradle just put all the projects, but had to denylist a couple:
```suggestion
    subprojects.findAll { !Set.of(
        project(':clients'),
        project(':clients:quarkus')
    ).contains(it)}.each {
        jacocoAggregation it
    }
```

---

Much better! Thanks for it! Updated.